Header header
int32 drone_id
int32 frame_id
int32 reference_frame_id

int64[] frame_ids
time[] frame_stamps
geometry_msgs/Pose[] poses
